backend/chainlit/data/__init__.py

```diff
@@ -462,7 +462,7 @@ async def update_thread(
 
 
 if api_key := os.environ.get("LITERAL_API_KEY"):
-    server = os.environ.get("LITERAL_SERVER")
+    server = os.environ.get("LITERAL_API_URL")
```

Comment:
we should be backward compatible with LITERAL_SERVER also

Reply:
good point, this would prevent a breaking change 👍

Reply:
fixed